build-system/log-module-metadata.js

```diff
@@ -14,23 +14,34 @@
  * limitations under the License.
  */
 
+/** @fileoverview Definitions of logging methods for transforms and linting. */
+
+/** @typedef {{variadic: boolean, messageArgPos: number}} */
+let LogMethodMetadataDef;
+
+const definitionFile = 'src/log.js';
+
+/** Functions exposed as singleton getters for `Log`. */
+const singletonFunctions = ['dev', 'user'];
+
 /**
+ * Known transformable logging methods.
  * @type {!Array<LogMethodMetadataDef>}
  */
-exports.transformableMethods = [
-  {name: 'assert', variadic: true, startPos: 1},
-  {name: 'assertString', variadic: false, startPos: 1},
-  {name: 'assertNumber', variadic: false, startPos: 1},
-  {name: 'assertBoolean', variadic: false, startPos: 1},
-  {name: 'assertEnumValue', variadic: false, startPos: 2},
-  {name: 'assertElement', variadic: false, startPos: 1},
-  {name: 'fine', variadic: true, startPos: 1},
-  {name: 'info', variadic: true, startPos: 1},
-  {name: 'warn', variadic: true, startPos: 1},
-  // Disable any of the error methods linting for now since these methods
```

Comment:
Enabled these since they don't break anything and compiled message templates from this methods would break otherwise.